src/hyperion/parameters/gridscan.py

```diff
@@ -94,6 +97,7 @@ def detector_params(self):
         ), "Detector distance must be filled before generating DetectorParams"
         os.makedirs(self.storage_directory, exist_ok=True)
         return DetectorParams(
+            detector_size_constants=self.detector,  # type: ignore
```

Comment:
Could: Can you put a comment here that this should get tidied by https://github.com/DiamondLightSource/hyperion/issues/1307